operations/helm/charts/mimir-distributed/templates/admin-api/admin-api-servmon.yaml

```diff
@@ -37,8 +37,10 @@ spec:
         - sourceLabels: [job]
           replacement: "{{ $.Release.Namespace }}/admin-api"
           targetLabel: job
-        - replacement: "{{ include "mimir.clusterName" $ }}"
+        {{- if .clusterLabel.enabled }}
+        - replacement: "{{ .clusterLabel.value | default (include "mimir.clusterName" $) }}"
```

Comment:
WDYT about moving this `if` inside the `mimir.clusterName` template? Would reduce repetition.

If you decide not to, then you will also need to update the `create-servmon` script to do this `if`.

Reply:
I'd rather remove that script and have a common template for the service monitors. 
I think including it in the `mimir.clusterName` would be misleading. I can image it in a `mimir.monitoringClusterName` , but maybe just having one place for it would be better. Let me try to make a single template out of this whole file.

Reply:
look better now